src/ray/raylet/scheduling/cluster_task_manager.h

```diff
@@ -115,6 +117,10 @@ class ClusterTaskManager {
 
   std::string DebugString() const;
 
+  /// Reiterate all local infeasible tasks and register them to task_to_schedule_ if it
```

Comment:
Can we keep this as a private method, and always invoke it at the end of SchedulePendingTasks()?

Reply:
In this case, I prefer to add some throttling logic here since `SchedulePendingTasks ` is called frequently. Do you have an objection to this idea?

Reply:
I don't think the added overhead here is high enough to justify the additional complexity.

Reply:
In fact, the overhead is likely exactly the same as the current implementation without the infeasible queue.

Reply:
But isn't it still worse than the overhead of the old scheduler (which schedules infeasible tasks only when new nodes are added)?

Also, just purely out of curiosity, but what's the standard of high overhead in raylet? If we have 100 infeasible tasks, this can probably add pretty big overhead given ScheduleAndDispatch is called for every node id for each heartbeat (if we have 100 nodes, we will have 100 * 100 iteration per 100 ms)